RNA_DNA_classes.py

```diff
@@ -0,0 +1,36 @@
+class Rna(str):
+    base_pairs = {'A': 'U', 'U': 'A',  'G': 'C', 'C': 'G', 'a': 'u', 'u': 'a',  'g': 'c', 'c': 'g'}
+
+    def __init__(self, sequence):
+        self.sequence = sequence
+        for base in self.sequence:
+            if base not in self.base_pairs:
+                raise ValueError('RNA sequence can contain only A, U, G and C bases')
+
+    def gc(self):
+        return (self.count('G') + self.count('C'))/len(self)
```

Comment:
It is necessary to consider lowercase symbols in `gc()` function + add multiplication by 100%